src/view/draggable/draggable.jsx

```diff
@@ -264,6 +264,7 @@ export default class Draggable extends Component<Props> {
     draggingOver: ?DroppableId,
   ): StateSnapshot => ({
     isDragging: (isDragging || isDropAnimating),
```

Comment:
I am not sure if we should keep this as true... I think we should keep the isDragging value the same (not changes required)